src/ScrollView/scrollView.scss

```diff
@@ -0,0 +1,113 @@
+@import '~gemini-scrollbar/gemini-scrollbar.css';
+@import '~sass-flex-mixins/src/sass-flex-mixins.scss';
+
+$scrollbarSize: 7px !default;
+$scrollbarSizeBig: 9px !default;
```

Comment:
down below, right? It's because when in `:hover` and `:active` state, the scrollbars is thicker.